### PR TITLE
Rotate and consolidate AWS keys

### PR DESCRIPTION
**Pull request by DwayneJengSage**
Part of https://sagebionetworks.jira.com/browse/BRIDGE-2887

Note that this is only the stopgap measure to get us through the next few days. In the next week or so, I will be planning to change Server code to use role-based access instead of user-based access.

This has been tested in Develop.

### CODE REVIEW

**Review by alxdarksage — APPROVED** [review]
It makes sense to me what you're doing, but I don't know enough to know where the AWSIAMBridgeServer2ServiceUserAccessKey and similar values are pulled from.

File: config/prod/bridgeserver2.yaml [hunk 1]
```diff
@@ -34,8 +26,6 @@ parameters:
   HibernateConnectionUsessl: 'true'
   NewRelicAppName: bridgeserver2-prod
   NewRelicLicenseKey: !ssm /infra/NewRelicLicenseKey
```
Comment (zaro0508) [inline config/prod/bridgeserver2.yaml:28]: aren't you going to setup roles to access newrelic and synapse keys as well?

**Review by zaro0508 — APPROVED** [review]
would be nice to setup roles to access newrelic and synapse keys as well.  otherwise it looks fine to me.

File: config/prod/bridgeserver2.yaml (hunk 1 above)
Comment (DwayneJengSage) [reply config/prod/bridgeserver2.yaml:28]: No, because those changes are neither urgent nor are they related to the AWS security issue.

**Pull request merged**